server/test/conftest.py

```
@@ -74,9 +74,12 @@ def start_server():
         if proc.poll() is not None:
             raise RuntimeError("server failed to start")
         try:
-            with urllib.request.urlopen(f"{BASE_URL}/health", timeout=5) as resp:
+            with urllib.request.urlopen(f"{BASE_URL}/model-version", timeout=5) as resp:
```

Ruff (0.14.1)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)